MountSionCBSSecondary/SchoolWebsite
# [WIP] adding policies and heading formatting

# Description

First pass at the policies page, pushing early to get feedback

## What

- Add a grid format as a landing page
- Adding some styling to the basic header

## Why

As part of the feature request the stakeholders wish to have school policies included on the website. This is the front/landing page of all school policies which can be linked to

## How

Using the Semantic styling available and hardcoding some of the text from each document

## Verification Steps

Build Locally and see that the layout page looks like the following image

![image](https://user-images.githubusercontent.com/6498727/40024735-2faa1ad0-57c7-11e8-9144-1d396cdd4f1a.png)


# Checklist:
